lib/cfg-block.c

```diff
@@ -61,6 +61,50 @@ cfg_block_format_name(CfgBlockGenerator *s, gchar *buf, gsize buf_len)
   return buf;
 }
 
+/* token block args */
+
+static void
+_validate_args_callback(gpointer k, gpointer v, gpointer user_data)
+{
+  CfgArgs *defs = ((gpointer *) user_data)[0];
+  const gchar *reference = ((gpointer *) user_data)[1];
+  gboolean *problem_found = ((gpointer *) user_data)[2];
+
+  if ((!defs || cfg_args_get(defs, k) == NULL))
+    {
+      if (cfg_args_is_accepting_varargs(defs))
+        {
+          msg_verbose("Unknown argument, adding it to __VARARGS__",
```

Comment:
My understanding is that here we cannot differentiate between valid driver options and invalid ones.
We can only know if the option is defined in the scl block or not.

I wonder how this part helps in debugging as if the user configures a lot of options and an invalid option too, he/she will see the parsing error anyway.

Reply:
This makes it possible to see that an argument was indeed not defined explicitly, so even if you have '...' as argument, you'll see (in debug mode) that an argument is using the wildcard mode.

I agree that it could trigger an error later on (when pasting __VARARGS__ into somewhere), but this message can be a useful when __VARARGS__ is pasted to somewhere where it is not actually validated.

Maybe this should be a debug message, instead of a verbose, I'll do that before merging.